brush-core/src/lib.rs

```diff
@@ -36,6 +36,9 @@ pub mod traps;
 pub mod variables;
 
 pub use arithmetic::EvalError;
+pub use brush_parser::{
```

Comment:
I left a more general comment about exposing these, but with that said, I'd lean toward any re-exports of `brush_parser` types be done in a child module (e.g., `brush_core::parser`).

Reply:
On that note, would it be possible to just export the entirety of `brush_parser` as a `parser` module? This would help to avoid having to list multiple crates in my `Cargo.toml` configuration, especially since the two crates are intertwined with each other.